### ISSUE TITLE
Don't suggest FAA0002 for xunit 3's Assert.Skip... usage

### ISSUE THREAD

**Issue by deMD** [fluentassertions/fluentassertions.analyzers#404]
Currently when using conditional tests with xUnit 3's Assert.Skip... I get the notification that I should use the Fluent Assertions alternative, however no such alternative exists.

```csharp
  [Fact]
  public void TestsSomething()
  {
      Assert.SkipWhen(true, reason: "skip this when true"); // <- this should not raise an analyzer FAA0002 issue.

      // Arrange
      // ...

      // Act
      var result = true;

     // Assert
    result.Should().BeFalse();
  }
```
